main/templates/admin/bit/mailgun.html

```diff
@@ -0,0 +1,10 @@
+{{
+  forms.panel_fields(
+    'Mailgun',
+    (form.mailgun_api_base_url, form.mailgun_api_key),
+    '''
+      Find the API key at
+      <a href="https://mailgun.com/app/domains/%s" target="_blank">https://mailgun.com/app/domains/%s</a>.
+    ''' % (request.host, request.host)
+    )
```

Comment:
the last parenthesis should be in the same line with `forms` :)

Thanks for reminding to check the rest.. #566